CONTRIBUTING.md

```diff
@@ -48,6 +48,16 @@ When submitting pull requests, make sure to do the following:
 - Remove trailing whitespace. Many editors will do this automatically.
 - Ensure any new files have [a trailing newline](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)
 
+## Feature Stages
```

Comment:
How about if there is a breaking change in the controller? How Agones is going to be versioned? v1.x.x -> v2.x.x?

Reply:
While I don't think it _likely_ there will be breaking change in the controller, we should probably mention we are following semver in the Agones releases. That sounds good?

Reply:
Should be good enough. This is a good example: https://kubernetes.io/docs/setup/release/version-skew-policy/

Reply:
Any change in the behavior that a user can get dependency on could be a breaking change, such as not setting finalizers on game servers.